WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController+Helper.swift

```diff
@@ -0,0 +1,145 @@
+import Foundation
+
+extension ReaderStreamViewController
+{
+
+    // A simple struct defining a title and message for use with a WPNoResultsView
+    public struct NoResultsResponse
+    {
+        var title:String
+        var message:String
+    }
+
+
+    /**
+    Returns the ReaderStreamHeader appropriate for a particular ReaderTopic
+    or nil if there is not one.  The caller is expected to configure the
+    returned header.
+
+    @param topic A ReaderTopic
+    @param An unconfigured instance of a ReaderStreamHeader.
+    */
+    public class func headerForStream(topic: ReaderTopic) -> ReaderStreamHeader? {
```

Comment:
Agreed, but I want to wait and include the test with the changes to the reader topic model.  Otherwise I'll just have to ditch one version and replace it with another.